taskcluster/kinds/merge-corpus/kind.yml

```diff
@@ -7,92 +7,93 @@ loader: taskgraph.loader.transform:loader
 
 transforms:
     - translations_taskgraph.transforms.worker_selection
-    - translations_taskgraph.transforms.find_upstreams:by_locales
     - taskgraph.transforms.task_context
+    - taskgraph.transforms.from_deps
+    - translations_taskgraph.transforms.fetches_from_upstream
     - taskgraph.transforms.run:transforms
     - translations_taskgraph.transforms.cached_tasks:transforms
     - taskgraph.transforms.task:transforms
 
 kind-dependencies:
-    - bicleaner
+    - post-bicleaner-dummy
     - toolchain
 
-task-defaults:
```

Comment:
Just curious, why move these defaults to tasks?

Reply:
Simply because there's only one entry in `task`. There's no benefit to splitting the entry. (In a much earlier iteration there was only one `merge` kind, and this was copied from that, which is why it had a `task-defaults` section.)